lib/takeaway.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'send_text'
+require_relative 'menu'
+
+class Takeaway
+  include SendText
+
+  attr_reader :menu, :basket
+
+  def initialize(menu_klass: Menu)
+    @basket = Hash.new(0)
+    @menu = menu_klass.new
+  end
+
+  def show_menu
+    menu.show
+  end
+
+  def order(item, qty = 1)
+    fail 'This item is not on the menu' unless available?(item)
+    basket[item] += qty
+    "#{item} x#{qty} added to your basket"
+  end
+
+  def basket_content
+    fail 'Basket is empty' if basket.empty?
+    basket.map { |item, qty| "#{item} x#{qty}: £#{qty * menu.dishes[item]}"}
+          .join("\n")
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver. (https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)<br>Style/MultilineOperationIndentation: Use 2 (not 6) spaces for indenting an expression spanning multiple lines.